migrations/versions/9a6d8b560a5a_.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""remove houston_config
+
+Revision ID: 9a6d8b560a5a
+Revises: 19dd4e2c1e63
+Create Date: 2022-03-18 20:47:33.020537
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+import app.extensions
+
+
+# revision identifiers, used by Alembic.
+revision = '9a6d8b560a5a'
+down_revision = '19dd4e2c1e63'
+
+
+def upgrade():
+    """
+    Upgrade Semantic Description:
+        ENTER DESCRIPTION HERE
+    """
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('houston_config', schema=None) as batch_op:
+        batch_op.drop_index('ix_houston_config_created')
+        batch_op.drop_index('ix_houston_config_indexed')
+        batch_op.drop_index('ix_houston_config_updated')
+
+    op.drop_table('houston_config')
```

Comment:
Wondering if we should print out all the data in case the table isn't empty?

Reply:
I suppose it wouldn't hurt. 😸